pkg/alerts/policies.go

```diff
@@ -137,3 +241,64 @@ type alertPolicyResponse struct {
 type alertPolicyRequestBody struct {
 	Policy Policy `json:"policy"`
 }
+
+type alertQueryPolicyCreateResponse struct {
+	QueryPolicy QueryPolicy `json:"alertsPolicyCreate"`
+}
+
+type alertQueryPolicyUpdateResponse struct {
+	QueryPolicy QueryPolicy `json:"alertsPolicyCreate"`
+}
+type alertQueryPolicyResponse struct {
+	Actor struct {
+		Account struct {
+			Alerts struct {
+				PoliciesSearch struct {
+					NextCursor *string        `json:"nextCursor"`
+					Policies   []*QueryPolicy `json:"policies"`
+				} `json:"policiesSearch"`
+			} `json:"alerts"`
+		} `json:"account"`
+	} `json:"actor"`
+}
+
+type alertQueryPolicyDeleteRespose struct {
+	AlertsPolicyDelete struct {
+		ID int `json:"id,string"`
+	} `json:"alertsPolicyDelete"`
+}
+
+const (
+	alertPolicyQueryPolicy = `query($accountID: Int!, $policyIDs: [ID!]!) {
+		actor {
+			account(id: $accountID) {
+				alerts {
+					policiesSearch(searchCriteria: {
+						ids: $policyIDs
+					}) {
+						policies {
+							id
+							name
+							incidentPreference
```

Comment:
Consider making this a `const` that is added to each of these other constants, so if suddenly there is a new field we have one place to update.